pages/treeherder.py

```diff
@@ -7,15 +7,20 @@
 from selenium.webdriver.support.ui import WebDriverWait as Wait
 
 import expected
+import random
```

Comment:
This still needs to be moved to the top to comply with PEP8 (unfortunately the linters do not pick up on this).